crates/searchers/openapi.yml

```diff
@@ -0,0 +1,646 @@
+      type: object
+      required:
+        - kind
+        - tokens
+        - sqrtPrice
+        - liquidity
+        - tick
+        - liquidityNet
+        - fee
+      properties:
+        kind:
+          type: string
+          enum: [concentratedliquidity]
+        tokens:
+          type: array
+          items:
+            $ref: "#/components/schemas/Token"
+        sqrtPrice:
+          $ref: "#/components/schemas/BigInt"
+        liquidity:
+          $ref: "#/components/schemas/TokenAmount"
+        tick:
+          $ref: "#/components/schemas/BigInt"
+        liquidityNet:
+          description: |
+            A map of tick indices to their liquidity values.
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/TokenAmount"
+        fee:
+          $ref: "#/components/schemas/Decimal"
+
+    ForeignLimitOrder:
+      description: |
+        A 0x-like limit order external to CoW Protocol.
+      type: object
+      required:
+        - kind
+        - hash
+        - makerToken
+        - takerToken
+        - makerAmount
+        - takerAmount
+        - takerTokenFeeAmount
+      properties:
+        kind:
+          type: string
+          enum: [limitorder]
+        hash:
+          $ref: "#/components/schemas/Digest"
+        makerToken:
+          $ref: "#/components/schemas/Token"
+        takerToken:
+          $ref: "#/components/schemas/Token"
+        makerAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        takerAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        takerTokenFeeAmount:
+          $ref: "#/components/schemas/TokenAmount"
+
+    LiquidityParameters:
+      oneOf:
+        - $ref: "#/components/schemas/ConstantProductPool"
+        - $ref: "#/components/schemas/WeightedProductPool"
+        - $ref: "#/components/schemas/StablePool"
+        - $ref: "#/components/schemas/ConcentratedLiquidityPool"
+        - $ref: "#/components/schemas/ForeignLimitOrder"
+
+    Liquidity:
+      description: |
+        On-chain liquidity that can be used in a solution. This liquidity is
+        provided to facilitate onboarding new solvers. Additional liquidy that
+        is not included in this set may still be used in solutions.
+      allOf:
+        - $ref: "#/components/schemas/LiquidityParameters"
+        - type: object
+          required:
+            - id
+            - address
+            - gasEstimate
+          properties:
+            id:
+              description: |
+                An opaque ID used for uniquely identifying the liquidity within
+                a single auction (note that they are **not** guaranteed to be
+                unique across auctions). This ID is used in the solution for
+                matching interactions with the executed liquidity.
+              type: string
+            address:
+              $ref: "#/components/schemas/Address"
```

Comment:
What would be the address of a `zeroex` limit order?

Reply:
To me, it would be the address of the [0x contract](https://etherscan.io/address/0xdef1c0ded9bec7f1a1670819833240f027b25eff) (the contract that gets called to execute the limit order on-chain).

Reply:
Added some note on what the `address` is (basically just saying that what address is actually specified is liquidity-dependent).